astropy/utils/xml/writer.py

```diff
@@ -168,8 +179,6 @@ def xml_cleaning_method(self, method="escape_xml", **clean_kwargs):
                 )
             import bleach
 
-            if clean_kwargs is None:
```

Comment:
Seems out of scope?

Reply:
Yeah, but also unreachable and not worth a PR by itself. Happy to revert it, I just couldn't resist scratching that itch.

Reply:
Maybe @taldcroft remembers why it was added in the first place in #4423 ?

Reply:
For a 9 years old PR I'm willing to bet the reason was python 2 🐍